packages/central-server/app/admin/patientMerge/mergePatient.js

```diff
@@ -26,7 +26,6 @@ export const simpleUpdateModels = [
   'DeathRevertLog',
   'UserRecentlyViewedPatient',
   'PatientProgramRegistration',
-  'PatientProgramRegistrationCondition',
```

Comment:
Since PatientProgramRegistrationCondition are now associated with Patients through PatientProgramRegistration they don't need to be merged directly.